### ISSUE TITLE
Buttons labeling for screen readers

### ISSUE THREAD

**Issue by tyrylu** [keybase/client#19355]
Hello.
When using the desktop app with the latest NVDA snapshot or the mobile app with VoiceOver i unfortunately found out that almost all the buttons aren't correctly labeled, they have no semantic information at all, actually. The same applies to the main navigation. For example joining the airdrop or authorizing a new device was basically a game of luck (thank goodness for the command line device add flow and some luck with the QR scanning). Yes, i know, this is basically a specific issue of #10370, but because that got exactly no response, i thought it wouldn't be as bad an idea to bump this one again.

**Comment by devil418** [comment]
Confirmed. Accessibility in Keybase is almost nonexistent. Fixing this would be appreciated, not only for me, but for all my friends who use KBFS a lot, but struggle with that horrible user interface.

**Comment by alex19EP** [comment]
yes it would be good to improve accessibility for the blind in all keybace products. ignoring these problems does not look very nice.
